DSE_6.9_Release_Notes.md

```diff
@@ -2,6 +2,26 @@
 DSE 6.9.x is compatible with Apache Cassandra&trade; 3.11 and adds additional production-certified changes, if any.
 Components that are indicated with an asterisk (&ast;) (if any) are known to be updated since the prior patch version.
 
+# Release notes for 6.9.12
+29 July 2025
```

Comment:
It seems the date it wrong, unless we are going to wait until 29th with the release.

Reply:
That date was autogenerated Dunno where it came from :shrug: Let me amend it

Reply:
I _think_ it comes from jira, where the dates could be wrongly autogenerated but I haven't checked the code.